CMakeLists.txt

```diff
@@ -606,7 +612,7 @@ if(DEMANGLE_DIR)
   target_compile_definitions(rccl PRIVATE "HAVE_CPLUS_DEMANGLE=1")
   target_compile_definitions(rccl PRIVATE "HAVE_DECL_BASENAME=1")
 endif()
-if(${hipcc_version_string} VERSION_GREATER_EQUAL "6.1.33591")
+if(${compiler_version_string} VERSION_GREATER_EQUAL "6.1.33591")
```

Comment:
@lawruble13 this one seems to be related to hipcc version... is there an equivalent for amdclang version?